README.md

```diff
@@ -60,7 +60,6 @@ If you use the
 add this to your `active_admin.sass` file:
 
 ```sass
-@import arctic_admin/base
```

Comment:
I think you meant to copy this line out, not cut.

Reply:
> I think you meant to copy this line out, not cut.

Sorry for my mistake, i've updated.